firmware/tunerstudio/rusefi.input

```diff
@@ -1854,9 +1856,14 @@ cmd_stop_engine              = "w\x00\x99\x00\x00"
 		field = "Injection Mode",						crankingInjectionMode
 		field = "Base Fuel Pulse Width", 				cranking_baseFuel
 		
+	dialog = crankingIAC, "IAC"
```

Comment:
Gather all IAC settings in a separate sub-panel 'IAC' inside 'Cranking Settings'